Proposed remove step 1

dmauser opened the pull request:
Proposed remove Step 1 which is not necessary. That can cause confusion as this certificate is a Public Commercial Certificate (DigiCert Global Root CA) and is not related for native Azure certificate authentication which requires only self sign PFX and Root CA certificates.

azurecla commented:
Hi __@rmaclean__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

mimig1 commented:
Thanks for the contribution rmaclean! I actually attempted to fix this a few days ago, then learned it's a known issue with the site caused by the NOTE. To get this fixed in the short term, I'll file a PR to remove the NOTE and put that text in #5. (Ross, you can accept this PR.)

dmauser committed:
Proposed remove step 1

Proposed remove Step 1 which is not necessary. That can cause confusion as this certificate is a Public Commercial Certificate (DigiCert Global Root CA) and is not related for native Azure certificate authentication which requires only self sign PFX and Root CA certificates.
